luigi/contrib/rdbms.py

```diff
@@ -27,7 +27,125 @@
 logger = logging.getLogger('luigi-interface')
 
 
-class CopyToTable(luigi.task.MixinNaiveBulkComplete, luigi.Task):
+class _MetadataColumnsMixin(object):
+    """Provide an additinal behavior that adds columns and values to tables
+
+    This mixin is used to provide an additional behavior that allow a task to
+    add generic metadata columns to every table created for both PSQL and
+    Redshift.
+
+    Example:
+
+        This is a use example or how this mixin could come handy and how to use
+        it.
+
+        .. code:: python
+
+            class CommonMetaColumnsBehavior(object):
+                def update_report_execution_date_query(self):
+                    query = 'UPDATE {0} ' \
+                            "SET date_param = DATE '{1}' " \
+                            'WHERE date_param IS NULL'.format(self.table, self.date)
```

Comment:
Can you use the same quote style for these lines? Rather than going back and forth between single and double quote.